packages/express/src/authenticateRequest.ts

```diff
@@ -0,0 +1,97 @@
+import type { RequestState } from '@clerk/backend/internal';
+import { AuthStatus, createClerkRequest } from '@clerk/backend/internal';
+import { handleValueOrFn } from '@clerk/shared/handleValueOrFn';
+import { isDevelopmentFromSecretKey } from '@clerk/shared/keys';
+import { isHttpOrHttps, isProxyUrlRelative, isValidProxyUrl } from '@clerk/shared/proxy';
+import type { Response } from 'express';
```

Comment:
❓ How is this different than the native `Response` class?

Reply:
It seems they are different types:
<img width="1462" alt="Screenshot 2024-04-04 at 22 24 55" src="https://github.com/clerk/javascript/assets/3936408/948ff802-b34b-4b7d-bbc6-392397a48ea8">